master/internal/db/postgres_trial.go

```diff
@@ -163,7 +163,8 @@ WHERE id = $1`, id, restartCount); err != nil {
 func (db *PgDB) fullTrialSummaryMetricsRecompute(
 	ctx context.Context, tx *sqlx.Tx, trialID int,
 ) error {
-	// DISCUSS: can we limit this to recompute only a single metric type?
+	// PERF (DET-9566): we can probably limit this to recompute only a single metric type and it would
```

Comment:
not sure `PERF` is a universally recognized thing?..

```suggestion
	// TODO(DET-9566): we can probably limit this to recompute only a single metric type and it would
```